src/rpc/server.cpp

```diff
@@ -314,6 +314,10 @@ static UniValue getrpcinfo(const JSONRPCRequest& request)
     UniValue result(UniValue::VOBJ);
     result.pushKV("active_commands", active_commands);
 
+    const std::string path = LogInstance().m_file_path.native().c_str();
```

Comment:
Could use `m_file_path.string()` instead. It might fix the issue.

Reply:
Thanks, it fixed it.